owasp-top10-2017-apps/a3/snake-pro/app/pass/pass.go

```diff
@@ -1,6 +1,25 @@
 package pass
 
+import (
+	"fmt"
+	"crypto/sha256"
+	"math/rand"
```

Comment:
Using `math/rand` is considered [insecure](https://github.com/securego/gosec/blob/f254cec60bf1482ae2cdfe1a72da83813ddfe336/rules/rand.go#L44) to generate random passwords. Would you mind using `crypto/rand` instead? 😅

Reply:
Done.